ocs_ci/utility/vsphere.py

```
@@ -1972,3 +1973,66 @@ def wait_for_vm_status(self, vm, desired_status, timeout=300, interval=10):
+            logger.info(f"Checking if storage policy exists: {policy_name}")
+            list_cmd = "govc storage.policy.ls"
+            result = exec_cmd(list_cmd, env=govc_env, timeout=60)
+            policies_output = result.stdout.decode("utf-8").strip().split("\n")
+
+            # govc output format: "UUID  policy-name"
+            # Check if the policy name exists in any line
+            policy_found = False
+            for policy_line in policies_output:
+                if policy_name in policy_line:
+                    policy_found = True
+                    logger.info(f"Found storage policy in line: {policy_line}")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)